### DISCUSSION TITLE
[Question] How to modify categories?

### DISCUSSION THREAD

**Discussion by RukshanJS** [ananthakumaran/paisa#160] [Q&A]
How can I modify categories like Food etc.? add new ones... Tried but couldn't find such an option.

**Comment by ananthakumaran** [comment]
What have you tried? You can start with the [accounts](https://paisa.fyi/reference/accounts/) documentation. There is not much context available to give a detailed answer. You need to understand the base model of paisa first, try to go through the [tutorial](https://paisa.fyi/getting-started/tutorial/)
